package.json

```diff
@@ -46,6 +46,8 @@
     "react-bootstrap": "^1.6.3",
     "react-dom": "^17.0.2",
     "react-icons": "^4.2.0",
+    "react-input": "^4.1.3",
```

Comment:
Is `react-input` required? I didn't see you import it anywhere. Is it a `peerDependency` of `react-select`?